cmd/ftl/cmd_deploy.go

```diff
@@ -100,10 +103,53 @@ func (d *deployCmd) Run(ctx context.Context, client ftlv1connect.ControllerServi
 		return err
 	}
 
+	if d.Wait {
+		logger.Infof("Waiting for deployment %s to become ready", resp.Msg.DeploymentName)
+
+		wg, ctx := errgroup.WithContext(ctx)
+		wg.Go(d.checkReadiness(ctx, client, resp.Msg.DeploymentName))
```

Comment:
I don't think you need any concurrency here, you can just run the for loop synchronously?

Reply:
very true 😃